tests/Utils/SchemaPrinterTest.php

```diff
@@ -148,6 +149,47 @@ public function testPrintNonNullArrayNonNullStringField() : void
         );
     }
 
+    /**
+     * @see it('Prints Field With "@deprecated" Directive')
+     *
+     * @dataProvider deprecationReasonDataProvider
+     */
+    public function testPrintDeprecatedField($deprecationReason, string $expectedDeprecationDirective) : void
+    {
+        $output = $this->printSingleFieldSchema([
+            'type' => Type::int(),
+            'deprecationReason' => $deprecationReason,
+        ]);
+        self::assertSame(
+            '
+type Query {
+  singleField: Int' . $expectedDeprecationDirective . '
+}
+',
+            $output
+        );
+    }
+
+    public function deprecationReasonDataProvider() : Generator
```

Comment:
Add return `Generator<string, array<mixed>>`

Reply:
when I add:

```php
    /**
     * @return Generator<string, array<mixed>>
     */
```

running `vendor/bin/phpcbf` removes it

Reply:
Ayright, let it be for now then